### DISCUSSION TITLE
Starting container by alias

### DISCUSSION THREAD

**Discussion by jasonvarga** [tighten/takeout#250] [Q&A]
Is it possible to start a container by its base alias?

For example, I typically only have one redis container. It would be nice to be able to do `takeout start redis` instead of needing to do `takeout start id-of-redis-container`, or `takeout start` and choose redis from the list.

If it's not possible, I'd be happy to try a PR if that sounds like a decent idea.

**Comment by josecanhelp** [comment]
Hey @jasonvarga, have you tried running `takeout start` by itself? You're provided with a menu of stopped containers that you can select from. I don't think there is a way to run `takeout start redis` and have that container start up. I think it's because it's possible to have multiple redis containers, but it would be a nice-to-have for that command to work if you have only one redis container stopped.

**Reply by jasonvarga** [reply]
Yeah I mentioned that's exactly what I do. I was just wondering if there was a shortcut. I guess not! Would you be open to a PR?

**Reply by josecanhelp** [reply]
A PR would be awesome.
